src/Microsoft.Health.Fhir.Api/Features/Routing/UrlResolver.cs

```diff
@@ -128,6 +129,32 @@ public Uri ResolveResourceUrl(ResourceElement resource, bool includeVersion = fa
             return new Uri(uriString);
         }
 
+        public Uri ResolveResourceWrapperUrl(ResourceWrapper resource, bool includeVersion = false)
```

Comment:
Suggest exacting a common private method and calling it instead of the duplication